src/v2/Apps/BuyerGuarantee/BuyerGuaranteeApp.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
+import { Footer } from "v2/Components/Footer"
+
+interface BuyerGuaranteeAppProps {}
+
+export const BuyerGuaranteeApp: React.FC<BuyerGuaranteeAppProps> = ({
+  children,
+}) => {
+  return (
+    <AppContainer>
+      <HorizontalPadding>
+        {children}
+        <Footer />
```

Comment:
Note that @The-Beez-Kneez is currently consolidating the footers in https://github.com/artsy/force/pull/7364 so this can be removed

Reply:
Nice!